packages/node_modules/@ciscospark/widget-meet/src/components/call-active/index.js

```diff
@@ -120,7 +125,7 @@ function ActiveCall({
         <div className={classNames(styles.waiting, 'participants-waiting')}>Waiting for others to join...</div>
       }
       {
-        localMediaStream && localMediaStream.active && (isSendingVideo || isSendingAudio) &&
+        hasVideo && localMediaStream && localMediaStream.active && (isSendingVideo || isSendingAudio) &&
```

Comment:
Wanted to keep it specific since I couldn't really come up with a use case where audio is not available. I think it makes sense to stick with `hasVideo` because this is a case where it's a state that shouldn't have local or remote video.